packages/api/cms-api/src/mailer/README.md

```diff
@@ -0,0 +1,139 @@
+# Mailer Module
```

Comment:
In my opinion it would be better to move this to the docs (perhaps under `Features / Modules -> MailerModule`) and just add a link to the docs in the readme